services/beacon.rb

```diff
@@ -23,7 +23,7 @@ def receive_alert
         alert_instance_id: payload[:incident_key],
         alert_instance_origination_time: payload[:trigger_time],
         description: settings[:description],
-        url: alert_link(payload[:alert][:id])
+        url: alert_link(payload.with_indifferent_access)
```

Comment:
I think this file can be removed altogether.

Reply:
heh I just searched for every `alert_link` and made the necessary changes, didn't even pay attention to the service name. I'll go ahead and remove all the beacon stuff.

Reply:
RIP ☠️